src/main/java/com/dev/cinema/model/User.java

```diff
@@ -0,0 +1,61 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String loginByEmail;
```

Comment:
check the UML and name this field accordingly please

Reply:
Ok.